airbyte-integrations/connectors/source-twitter/source_twitter/twitter.yaml

```diff
@@ -0,0 +1,50 @@
+version: "0.1.0"
+
+definitions:
+  selector:
+    extractor:
+      field_pointer: []
+  requester:
+    url_base: "https://api.twitter.com/2/tweets"
+    http_method: "GET"
+    authenticator:
+      type: "BearerAuthenticator"
+      header: "apikey"
+      api_token: "{{ config['api_key'] }}"
+    request_options_provider:
+      request_parameters:
+        query    : "{{ config['query'] }}" 
+  retriever:
+    record_selector:
+      $ref: "*ref(definitions.selector)"
+    paginator:
+      type: DefaultPaginator
+      pagination_strategy:
+        type: "CursorPagination"
+        cursor_value: "{{ response.meta.next_token }}"
+        stop_condition: "{{ 'next_token' not in response['meta'] }}"
+        page_size: 100
+      page_size_option:
+        field_name: "max_results"
+        inject_into: "request_parameter"
+      page_token_option:
+        field_name: "next_token"
+        inject_into: "request_parameter"
+      url_base: "*ref(definitions.requester.url_base)"
+    requester:
+      $ref: "*ref(definitions.requester)"
+  base_stream:
+    retriever:
+      $ref: "*ref(definitions.retriever)"
+
+  recent_search_tweets_stream:
+    $ref: "*ref(definitions.base_stream)"
+    $options:
+      name: "recent_search_tweets"
+      path: "/search/recent"
```

Comment:
Rename to tweets_stream,  I think you must implement a DataStreamSlicer because twitter can process a gigantic quantity of data and have the incremental sync method is a must.

Reply:
@marcosmarxm  Renamed & added Date Stream slicer. Can you please review the connector?